c2h/include/c2h/extended_types.h

```diff
@@ -45,6 +45,14 @@
 #  endif
 #endif // TEST_BF_T
 
+#ifndef TEST_INT128
+#  if _CCCL_HAS_INT128() && !_CCCL_CUDA_COMPILER(CLANG) // clang-cuda crashes with int128 in generator.cu
+#    define TEST_INT128() 1
+#  else
+#    define TEST_INT128() 0
+#  endif
+#endif // TEST_INT128
```

Comment:
If we introduce `TEST_INT128`, we may want to replace all uses of `_CCCL_HAS_INT128` in CUB unit tests by the new macro. Otherwise, it seems inconsistent. What do you think?